src/TwigComponent/tests/Fixtures/templates/components/parent_form_component.html.twig

```diff
@@ -0,0 +1,5 @@
+<div>
+    {{ component('textarea_component', {
+        'data-model': 'value:content'
```

Comment:
~This should work w/o `value:`, right? Is this tested?~

I see it's unit tested.